swagger-spec/registrations/definition.yaml

```diff
@@ -141,6 +144,19 @@ properties:
         type: string
         readOnly: true
         description: 'The reasoning for why this registration was withdrawn, as entered by the administrator that initiated the withdrawal.'
+      draft_registration:
+        type: string
+        readOnly: false
+        description: 'The unique ID of the draft registration from which the registration was created.'
```

Comment:
Rewording suggestion: The ID of the draft registration from which the registration will be created.